report-ng/app/package.json

```diff
@@ -9,6 +9,7 @@
     ],
     "license": "Apache-2.0",
     "dependencies": {
+        "@aurelia-mdc-web/button": "^7.1.3",
```

Comment:
Button has been added, but not used in `threads.html`

Reply:
`mdc-button` fragments have now been removed.